app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
According to the code style checklist, when you import multiple classes/modules, you should use parentheses. It would look like this:

python
from app.errors import (
    VaccineError, 
    NotWearingMaskError
)
```